### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by longieirl**
Correct href links.

### CODE REVIEW

**Comment by longieirl** [comment]
@theStiggySpd please review. Thanks.

File: cap/README.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ The SAP Cloud Application Programming Model (CAP) is a framework of languages, l
 
 This repository provides sample CAP projects using different router configurations and the steps required to deploy to Cloud Foundry;
 
-1. [Managed Approuter](/cap-fiori-mta/README.md) with SAP Hana Cloud service
+1. [Managed Approuter](../cap/cap-fiori-mta/README.md) with SAP Hana Cloud service
 
-2. [Standalone Approuter](/cap-fiori-mta-standalone/README.md) with an inmemory database
+2. [Standalone Approuter](../cap/cap-fiori-mta-standalone/README.md) with an inmemory database
```
Comment (theStiggySpd) [inline cap/README.md:13]: inmemory `->` in-memory

**Review by theStiggySpd — CHANGES_REQUESTED** [review]
small change requested

**Review by theStiggySpd — APPROVED** [review]
changes looks good now

**Pull request merged**